docs/figma/ios_prototype_v2/styles.css

```
@@ -949,3 +965,441 @@ body {
+  background: rgba(255, 255, 255, 0.04);
+  box-shadow: inset 0 0 0 10px rgba(255, 255, 255, 0.12);
+}
+
+.plate-wheel-description {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  border: 0;
+  clip: rect(0 0 0 0);
```

Stylelint (17.4.0)

[error] Unexpected deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)